DOCO-920 Autopilot Custom Scheduling (#8263)
pantheon-systems/documentation: source/partials/autopilot/custom-scheduling.md

---
contenttype: [partial]
categories: [update, test]
cms: [--]
product: [autopilot]
integration: [--]
tags: [--]
reviewed: "2023-03-17"
---

> If you are on the Platinum or Diamond plan, you have access to Custom Scheduling. This functionality allows you to schedule Autopilot to run on a more specific cadence. For example, on a particular day of the week, or day of the month.

Reviewer: ```suggestion
Custom Frequency Scheduling is available to sites owned by Platinum and Diamond Workspaces. This functionality allows you to schedule Autopilot to run on a more specific cadence. For example, on a particular day of the week, or day of the month.
```

> Custom Frequency Scheduling is available to sites owned by Platinum and Diamond Workspaces. This functionality allows you to schedule Autopilot to run on a more specific cadence. For example, on a particular day of the week, or day of the month.

Reviewer: Since this appears directly below the note about Custom Scheduling only being for Platinum and Diamond workspaces, this note feels a little repetitive. Maybe we remove the first sentence and condense it to say "Custom Frequency Scheduling allows you to schedule Autopilot..." ?

Author: Done!

To update your settings, navigate to your site's **Configuration** tab, scroll down to the **Destination & Frequency** section, and select **Custom** under the **Update Frequency** section.

<Alert title="Note"  type="info" >

> Please note that the day you select for Autopilot to run is when updates will **start** running, and does not mean the update will be complete on that day. For best results, schedule Autopilot to run the day before you plan to review test results.

Reviewer: ```suggestion
Please note that the day you select for Autopilot to run is when updates will **start** running, not necessarily when they will be completed. For best results, schedule Autopilot to run the day before you plan to review updates.
```
